config/jobs/kubernetes/sig-cloud-provider/aws/kops/kops-periodics.yaml

```diff
@@ -203,6 +203,36 @@ periodics:
   annotations:
     testgrid-dashboards: google-aws, sig-cluster-lifecycle-kops
     testgrid-tab-name: kops-aws-imageubuntu1604
+- interval: 4h
+  name: ci-kubernetes-e2e-kops-aws-containerd
+  labels:
+    preset-service-account: "true"
+    preset-aws-ssh: "true"
+    preset-aws-credential: "true"
+  spec:
+    containers:
+      - args:
+        - --timeout=140
+        - --bare
+        - --scenario=kubernetes_e2e
+        - --
+        - --cluster=e2e-kops-aws-containerd.test-cncf-aws.k8s.io
+        - --deployment=kops
+        - --env=KUBE_SSH_USER=admin
+        - --extract=ci/latest
+        - --ginkgo-parallel
+        - --kops-args="--image=136693071363/debian-10-amd64-20191117-80 --container-runtime=containerd"
```

Comment:
Is there a reason this is a specific debian image? Any reason to choose this over Kops' default images? Just curious.

Reply:
I just did some testing with e2e for containerd, so I can give better answer.
The tests work with the default images with a bunch of tweaks. The 2 main issues are that:
* the `dbus` package is not installed, but is needed by Protokube
* Docker is installed and runs by default (wouldn't expect this in general)

Reply:
For reference, the work to get started with e2e for containerd:
https://github.com/kubernetes/kops/pull/8175

```
   3/4854 Tests Failed.
 696/4854 Tests Passed!
4155/4854 Tests Skipped.
```

Reply:
Thinking a bit more, Debian 10 still has problems with iptables-nft and works only with special flag for Calico.
I updated the PR to use RHEL8 and Calico, which should work out of the box.
I don't have a problem using the default Kops images, but first would have to address the issues I noticed during the e2e test.